src/docfx/restore/RestoreMap.cs

```diff
@@ -12,100 +12,8 @@ namespace Microsoft.Docs.Build
 {
     internal static class RestoreMap
```

Comment:
>RestoreMap [](start = 26, length = 10)

Only restore-file-related logic in this class, would it be better to merge it with RestoreFile?

Reply:
i will change `restore file` part in another pr